screwdriver/screwdriver.go

```diff
@@ -439,18 +441,27 @@ func (a api) UpdateBuildStatus(status BuildStatus, meta map[string]interface{},
 		return fmt.Errorf("creating url: %v", err)
 	}
 
+	stats := map[string]interface{}{}
+	nodeId := os.Getenv("NODE_ID")
```

Comment:
is NODE_ID guaranteed to be available in all k8s runtimes ?

Reply:
Yes. `NODE_ID` is defined by [pod.yaml.hbs](https://github.com/screwdriver-cd/executor-k8s/blob/f6c039b5a84d5b4f1e7db16aaf8633e98a0a7d6b/config/pod.yaml.hbs#L43-L46) .
Thus if your `pod.yaml.hbs` contains this setting, `hostname` will be set correctly.